docs/learn/avalanche/avalanche-platform.md

```diff
@@ -3,33 +3,35 @@ description: Avalanche features 3 built-in blockchains which includes Exchange C
 sidebar_label: The Primary Network
 ---
 
-# Avalanche Platform
+# The Primary Network
 
 Avalanche is a heterogeneous network of blockchains. As opposed to homogeneous networks, where
 all applications reside in the same chain, heterogeneous networks allow separate chains to be
 created for different applications.
 
-The Primary Network is a special [Subnet](subnets-overview.md) that contains all validators
-(including validators of any custom Subnets). A node can become a validator for the Primary
-Network by staking at least 2,000 AVAX.
+The Primary Network is a special [Subnet](subnets-overview.md) that runs three blockchains:
+
+- The Contract Chain [(C-Chain)](avalanche-platform.md#contract-chain-c-chain)
+- The Platform Chain [(P-Chain)](avalanche-platform.md#platform-chain-p-chain)
+- The Exchange Chain [(X-Chain)](avalanche-platform.md#exchange-chain-x-chain)
+
+A node can become a validator for the Primary Network by staking at least **2,000 AVAX**.
 
 ![Primary network](/img/primary-network.png)
 
-All validators as members of the Primary Network are required to validate and secure the
-following 3 blockchains:
-the [**Exchange Chain (X-Chain)**](#exchange-chain-x-chain),
-the [**Platform Chain (P-Chain)**](#platform-chain-p-chain), and
-the [**Contract Chain (C-Chain**)](#contract-chain-c-chain).
+## The Chains
+
+All validators of the Primary Network are required to validate and secure the following:
 
-## Contract Chain (C-Chain)
+### Contract Chain
```

Comment:
I'd prefer to keep (C-Chain) in there since we pretty much never call it the "Contract Chain". Same for X-Chain and P-Chain.